### PR TITLE
Create ATO documnetation

### PR DESCRIPTION
**Pull request by stvnrlly**

### CODE REVIEW

File: .about.yml [hunk 1]
```diff
@@ -2,63 +2,112 @@
...
 # Who is the partner for your project? (Use the full name of the partner documented here: https://github.com/18F/dashboard/blob/staging/_data/partners.yml)
 partners:
-- 
+-
 
 # The main point of contact(s) and/or the issue reporting system for your project, and either a `mailto:` link or URL for each contact.
 # Items:
 # - url: URL for the link
 #   text: Anchor text for the link
 contact:
```
Comment (adelevie) [inline .about.yml:100]: Does this need to filled in?

File: .about.yml [hunk 2]
```diff
@@ -69,15 +118,15 @@ parent:
...
+-
 
 # What tags does your organization's blog associate with your project? You can find a list of 18F blog tags here: https://18f.gsa.gov/tags/
 blogTag:
-- 
+-
 
 # What technologies are used in this project?
 stack:
--
```
Comment (adelevie) [inline .about.yml:80]: Does this need to be filled in?

File: .about.yml [hunk 3]
```diff
@@ -86,15 +135,15 @@ stack:
 #   url: URL for detailed information
 #   badge: URL for the status badge
 services:
--
```
Comment (adelevie) [inline .about.yml:89]: fill in?

**Comment by stvnrlly** [comment]
Before this is merged, we should actually implement some of the referenced controls.
- [x] Set up New Relic monitoring
- [x] Set up `bandit` scanning as part of Travis CI
- [x] ~~After the `pep8` → `pycodestyle` transition, set up style linting on Travis CI~~ Taken care of on Code Climate

File: .about.yml (hunk 1 above)
Comment (stvnrlly) [reply .about.yml:100]: That's not a required field for `.about.yml`, but it doesn't hurt to fill it in. To my knowledge, the only thing that we use `.about.yml` for is the dependency license documentation.

File: .about.yml (hunk 3 above)
Comment (stvnrlly) [reply .about.yml:89]: This is also covered in the `component.yaml`, so I'd rather not have to maintain it in two places if it can be avoided.

**Pull request merged**